Aptoide/aptoide-client-v8
# [MOB-320] task: update material component lib version

**What does this PR do?**

   Update material component version  in order to fix cutting label in BN (in PT)

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Open app in Portuguese (don't forget translations) and check if Pesquisar label is cut.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-320](https://aptoide.atlassian.net/browse/MOB-320)

**Code Review Checklist**
